Userland/Utilities/image.cpp

```diff
@@ -41,6 +44,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto frame = TRY(decoder->frame(0)).image;
     Optional<ReadonlyBytes> icc_data = TRY(decoder->icc_data());
 
+    if (strip_color_profile)
```

Comment:
Does the decoder need it's `icc_data` extracted? Or can we just skip extracting it?
```c++
Optional<ReadonlyBytes> icc_data;
if (!strip_color_profile)
    icc_data = TRY(decoder->icc_data());
```

Reply:
(see below)